src/main/java/com/example/capstone3/Account.java

```diff
@@ -0,0 +1,80 @@
+package com.example.capstone3;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "Accounts")
+public class Account {
+
+    @Id
+    @Column(name = "account_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long accountId;
+    private String name;
+    private String accountType;
+
+       public Account() {
+    }
+
+    public Account(Long accountId, String name, String accountType, Customer customer, String status) {
+        this.accountId = accountId;
+        this.name = name;
+        this.accountType = accountType;
+        this.customer = customer;
+        this.status = status;
+    }
+
+    @ManyToOne
+    @JoinColumn(name = "customer_id")
+    private Customer customer;
+
+
+    @Column(columnDefinition = "VARCHAR(10)")
+    private String status;
+
+
+    public Long getAccountId() {
+        return this.accountId;
+    }
+
+    public void setAccountId(Long accountId) {
+        this.accountId = accountId;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getAccountType() {
+        return this.accountType;
+    }
+
+    public void setAccountType(String accountType) {
+        this.accountType = accountType;
+    }
+
+    public Customer getCustomer() {
+        return this.customer;
+    }
+
+    public void setCustomer(Customer customer) {
+        this.customer = customer;
+    }
+
+    public String getStatus() {
+        return this.status;
+    }
+
+    public void setStatus(String status) {
+        this.status = status;
+    }}
```

Comment:
enforce status checking here as well. There is an [article](https://www.baeldung.com/jpa-persisting-enums-in-jpa) about persisting enums. you should take a look at that and implement accordingly